wdio.conf.sapp.ios.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+const { sappBaseConfig } = require('./wdio.conf.sappbase');
+
+exports.config = {
+    ...sappBaseConfig,
+    capabilities: [
+        {
+            platformName: 'iOS',
+            'appium:autoWebview': true,
+            'appium:deviceName': 'iPhone 12',
+            'appium:app': '<path to iOS test app>',
```

Comment:
nit: add a TODO if there's an action that needs to be done to replace it with the correct path